BigTuna/New Group/FontExtension.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  FontExtension.swift
+//  BigTuna
+//
+//  Created by James Wu on 3/31/19.
+//  Copyright © 2019 aomori nebuta. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+extension UILabel {
+    // Based on https://stackoverflow.com/a/47307499
+    @objc var substituteFontName : String {
```

Comment:
looks like adding this in caused some font issues on the login page:
![image](https://user-images.githubusercontent.com/9666592/55378446-d9c36180-54cd-11e9-831f-95d450316909.png)

Reply:
Yea... i'm thinking we may need another PR entirely to handle fonts + styling !! Seems like it may be a bigger effort.

Reply:
Sweet, seems like we just had to set a single UILabel property. I believe it's fixed now 😄